test/run

```diff
@@ -2,10 +2,12 @@
 
 set -eu
 
-# run static code checks like pyflakes and pep8
-PYEXEFILES="$(git grep -lI '^#!.*python') $(git ls-files "*.py")"
-flake8 $PYEXEFILES
+if [ -z "${SKIP_STATIC_CHECK:-}" ]; then
+	# run static code checks like pyflakes and pep8
+	PYEXEFILES="$(git grep -lI '^#!.*python') $(git ls-files "*.py")"
+	flake8 $PYEXEFILES
 
-mypy --no-error-summary --ignore-missing-imports --scripts-are-modules .
+	mypy --no-error-summary --ignore-missing-imports --scripts-are-modules .
```

Comment:
Please squash this into the previous commit, or reorder them to do the "skip static checks" one first (which is cleaner).